storage/google_cloud/google_cloud_utils.py

```diff
@@ -0,0 +1,30 @@
+from urllib.parse import urlparse
+
+from google.cloud import storage
+
+
+def download_blob_to_string(bucket_credentials_file_path, blob_url):
+    """
+    Downloads the contents of a Google Cloud blob to a string.
```

Comment:
s/Google Cloud/Google Cloud Storage/ 

(Google's platform has multiple different 'blobs' - this one is a GCS blob)

Reply:
changed